tests/foreman/maintain/test_maintenance_mode.py

```diff
@@ -92,6 +105,14 @@ def _finalize():
         data_yml = yaml.safe_load(f)
     assert len(enable_sync_ids) == len(data_yml[':default'][':sync_plans'][':disabled'])
 
+    if local_advisor_enabled:
+        assert 'All timers stopped' in result.stdout
+        result = sat_maintain.execute('systemctl status iop-service-vuln-vmaas-sync.timer')
+        assert result.status == 3
+        assert 'Active: inactive (dead)' in result.stdout
+        result = sat_maintain.execute('systemctl list-timers')
+        assert 'iop-service-vuln-vmaas-sync.service' not in result.stdout
```

Comment:
What is this asserting? The fact that nothing triggers sync.service right now?

Reply:
Wrong assertion, I wanted to do `iop-service-vuln-vmaas-sync.timer`. Good catch:)